robocop/checkers/misc.py

```diff
@@ -335,7 +356,42 @@ def visit_Keyword(self, node):  # noqa
                     return_from = True
         if keyword_after_return:
             token = return_setting_node.data_tokens[0]
-            self.report("keyword-after-return", error_msg=error, node=token, col=token.col_offset + 1)
+            self.report(
+                "keyword-after-return",
+                error_msg=error,
+                node=token,
+                col=token.col_offset + 1,
+                end_col=token.end_col_offset + 1,
+            )
+        self.generic_visit(node)
+
+    visit_If = visit_For = visit_While = visit_Try = visit_Keyword
+
+
+class UnreachableCodeChecker(VisitorChecker):
+    """Checker for unreachable code after RETURN, BREAK or CONTINUE statements."""
+
+    reports = ("unreachable-code",)
+
+    def visit_Keyword(self, node):  # noqa
+        statement_node = None
+
+        for child in node.body:
+            if (ReturnStatement and isinstance(child, ReturnStatement)) or isinstance(child, Break) or isinstance(child, Continue):  # type: ignore[arg-type]
```

Comment:
``Break`` and ``Continue`` can be ``None`` if there is import error (RF<5). But maybe, since the "unreachable-code" works only for RF >=5, we don't need to worry about it and simplify the condition like proposed (it needs to be tested on different versions though).

```suggestion
           if isinstance(child, (ReturnStatement, Break, Continue):
```